modules/phpfpm/client.go

```diff
@@ -88,3 +126,34 @@ func (c client) fetchStatus(req *http.Request) (*status, error) {
 
 	return s, nil
 }
+
+func (c socketClient) Status() (*status, error) {
+	socket, err := fcgiclient.Dial("unix", c.socket)
```

Comment:
does this library support timeout?

Reply:
Yes, by Using the function DialTimeout acts like Dial but takes timeout as a third argument.

Reply:
lets use it then